dist/README.md

```diff
@@ -0,0 +1,27 @@
+# Leaderboard-list-app
+Demonstrate the use of Callbacks, Promises, ES6, Webpack and Modular JS
+### Objectives
+-  Use callbacks and promises.
+-  Learn how to use proper ES6 syntax.
+-  Use ES6 modules to write modular JavaScript.
+-  Use webpack to bundle JavaScript.
+## Milestone Reached
+ [x] Create an `npm project` with webpack.
+
+ [x] Write plain `HTML markup` with minimum styling
+
+ [x] Use `ES6` modules, with `import and export`.
```

Comment:
# Please make your README file looks professional: 
- It should contain all that's needed for running the app with the API(s) it consumes.
- It should have instructions about where and how to generate any necessary keys and credentials ...

- [ ] **Example**: 
- In Getting started section of the ReadMe: Kindly provide the link to the repo which needs to be cloned on the local Terminal
- How to make the project run on the local machine(Install Webpack). Also, talk about the installation of Packages.Json